packages/integration-karma/test/component/spellcheck-attribute/index.spec.js

```diff
@@ -0,0 +1,50 @@
+import { createElement } from 'lwc';
+
+import Container from 'x/container';
+
+function getSpellcheckValue(container, selector) {
+    const elm = container.shadowRoot.querySelector(selector);
+
+    return elm.getAttribute('spellcheck');
+}
+
+describe('spellcheck attribute', () => {
+    it('should cast to true static values other than false', () => {
+        const elm = createElement('x-container', { is: Container });
+        document.body.appendChild(elm);
+
+        return Promise.resolve().then(() => {
```

Comment:
Don't need to wait for the next tick to do the assertions, it should be synchronous.

Reply:
im asserting on the value of the reflected attribute. i dont need to wait another tick for the component to rerender?

Reply:
We should be seeing the correct values on initial render.